src/utils.js

```diff
@@ -341,3 +364,45 @@ export const prepareBudgetData = (budget) => (
         }]
     }), {})
 )
+
+/**
+ * validate score value and prepare data in baseline format
+ * @param  {Object}
+ */
+export const validateJankinessValue = (scoreValue) => {
+    let parsedScoreValue = scoreValue
+
+    if (!Array.isArray(scoreValue)) {
+        try {
+            parsedScoreValue = JSON.parse(scoreValue)
```

Comment:
for `--jankiness [50,100]` as CLI params, `argv.jankiness` has stringify value `"[50,100]"`, for single number `--jankiness 50`,  parsing is not required.

Reply:
One more thing, we have implemented it in same way as `budget`,
- single value is `max.` threshold.
- and array is `[min, max]`.

Reply:
Can we add a comment then that we expect the cli param to be a string here containing either `"[50,100]"` or just `"50"`. Also can we add an example in the readme because it doesn't say that the parameter has to be formatted as `-jankiness [50,1]`